src/airflow_dags/plugins/callbacks/slack.py

```diff
@@ -40,22 +40,24 @@ def slack_message_callback(message: str) -> list[BaseNotifier]:
         ),
     ]
 
-def _build_default_message(task_link: str, flag: str, urgency: Urgency) -> str:
+def _build_message(task_link: str, flag: str, urgency: Urgency, additional_message_context: str = "") -> str:
     """Return a sensible default message for the given urgency."""
     if urgency == Urgency.CRITICAL:
         return (
-            f"❌{flag} The {task_link} failed. "
+            f"❌{flag} The {task_link} failed. " +
+            additional_message_context +
             "Please see run book for appropriate actions."
         )
     return (
-        f"⚠️{flag} The {task_link} failed, but it's ok. "
+        f"⚠️{flag} The {task_link} failed, but it's ok. " + 
+        additional_message_context +
         "No out of hours support is required."
```

Comment:
```suggestion
        " No out of hours support is required."
```

Be tempted to put a space in here in case people don't put a trailing space in the additional message context.